generator/testdata/rust/gclient/golden/iam/v1/.sidekick.toml

```diff
@@ -0,0 +1,15 @@
+[General]
+SourceFormat = 'protobuf'
```

Comment:
nit: I think we add some toml struct tags so casing is a little more consistent. For examples see: https://pkg.go.dev/github.com/pelletier/go-toml/v2#example-Marshal-Commented

Reply:
Good idea. I made the command-line options and TOML entries have the same names too.